frontend/src/metabase/visualizations/types/click-actions.ts

```diff
@@ -167,5 +167,5 @@ export interface QueryClickActionsMode {
 
   clickActions: LegacyDrill[];
   fallback?: LegacyDrill;
-  ModeFooter?: (props: ModeFooterComponentProps) => JSX.Element;
+  ModeFooter?: (props: ModeFooterComponentProps) => JSX.Element | null;
```

Comment:
Since the type in `frontend/src/metabase/visualizations/click-actions/components/TimeseriesModeFooter.tsx` also has changed.